static/app/views/settings/account/accountIdentities.tsx

```diff
@@ -64,7 +64,7 @@ function IdentityItem({identity, onDisconnect}: IdentityItemProps) {
         </IdentityText>
       </InternalContainer>
       <InternalContainer>
-        <TagWrapper>
+        <TagWrapper css={{gap: space(0.75)}}>
```

Comment:
Why not add the property to the `TagWrapper` styled component?

Reply:
oops i totally missed that it's a styled component, just moved it